src/app/components/planner/planner.component.scss

```diff
@@ -435,6 +441,8 @@
 }
 
 /* Media queries */
+
+/* Resizes font size of date for smaller screens/browsers */
```

Comment:
I don't know if the comments are necessary. It adds a bit of clutter and is hopefully pretty straightforward (class name `.date-display` is set to smaller font-weight on a different screen size)